server/src/main/java/org/apache/iotdb/db/query/pool/RawQueryReadTaskPoolManager.java

```diff
@@ -40,10 +40,7 @@ public class RawQueryReadTaskPoolManager extends AbstractPoolManager {
   private static final Logger LOGGER = LoggerFactory.getLogger(RawQueryReadTaskPoolManager.class);
 
   private RawQueryReadTaskPoolManager() {
-    int threadCnt =
-        Math.min(
-            Runtime.getRuntime().availableProcessors(),
-            IoTDBDescriptor.getInstance().getConfig().getConcurrentSubRawQueryThread());
+    int threadCnt = IoTDBDescriptor.getInstance().getConfig().getConcurrentSubRawQueryThread();
```

Comment:
May lead to an IllegalArgumentException while initializing ThreadPoolExecutor run/debug within IDEA, but packaged jar works as expected.